modules/nrf_wifi/os/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ target_compile_definitions(
   $<$<BOOL:${CONFIG_NRF_WIFI_MGMT_BUFF_OFFLOAD}>:NRF_WIFI_MGMT_BUFF_OFFLOAD>
```

Comment:
What is the point of this last commit, can't you squash it to the first one?

Reply:
autosquash did not do this due to saucetag. Fixed now, Thanks!